README.md

```diff
@@ -96,9 +96,9 @@ for its support details.
 | CephFS | Dynamically provision, de-provision File mode RWO volume  | Beta           | >= v1.1.0          | >= v1.0.0        | Nautilus (>=14.2.2)  | >= v1.14.0         |
 |        | Dynamically provision, de-provision File mode RWX volume  | Beta           | >= v1.1.0          | >= v1.0.0        | Nautilus (>=v14.2.2) | >= v1.14.0         |
 |        | Dynamically provision, de-provision File mode ROX volume  | Alpha          | >= v3.0.0          | >= v1.0.0        | Nautilus (>=v14.2.2) | >= v1.14.0         |
-|        | Creating and deleting snapshot                            | Beta           | >= v3.1.0          | >= v1.0.0        | Octopus (>=v15.2.3)  | >= v1.17.0         |
-|        | Provision volume from snapshot                            | Beta           | >= v3.1.0          | >= v1.0.0        | Octopus (>=v15.2.3)  | >= v1.17.0         |
-|        | Provision volume from another volume                      | Beta           | >= v3.1.0          | >= v1.0.0        | Octopus (>=v15.2.3)  | >= v1.16.0         |
+|        | Creating and deleting snapshot                            | Beta           | >= v3.1.0          | >= v1.0.0        | Octopus (>=v15.2.4)  | >= v1.17.0         |
```

Comment:
@Madhu-1 while we started to support snapshot and clone on above Octopus version , so we added this documentation, so I am wondering how could that broken in that specific version ie `15.2.3`

Reply:
@humblec can you please test with the octopus 15.2.3 and check it's working? if its working am happy to close this PR and linked issue

Reply:
@Madhu-1 `subvolume info ` to `15.2.` version was added by https://tracker.ceph.com/issues/45180 , in your testing output it said, no handler found against the same Ceph Version which definitely should not be the case.   

```
E1221 05:44:29.034258       1 utils.go:185] ID: 25 Req-ID: pvc-f93d7ac0-3f9e-4551-9171-2636d671078c GRPC error: rpc error: code = Internal desc = API call not implemented server-side: No handler found for 'fs subvolume snapshot info'
```

@kotreshhr can you please confirm or share your thought. 

Additionally let me also test if possible.

Reply:
ah.. in the referenced issue it is looking for  `subvolume snapshot info` and not `subvolume info` .  so atleast in the initial implementaion it was only `subvolume info` we required and later `snapshot info` was added which added dependency to ceph version 15.2.4 may be.. have to cross check

Reply:
not really , initial implementation made use of `snapshot info` https://github.com/ceph/ceph-csi/pull/394/files#diff-5a0bf2f4254068588d0f2d432acb4cb68371c01290036f70f0b7df0b9da8f8b7R467 too.. have to confirm it was to `just to verify the snapshot` or an additional check it was.

Reply:
I don't think the above statement is correct. in the initial version snapshot info was added PTAL https://github.com/ceph/ceph-csi/pull/394/files#diff-e2d8af0e47a4dfb1dab79c4f16b787abbf3a5f94ebf4e8a9d5f41805246666f5R107-R138

Reply:
https://tracker.ceph.com/issues/46001 tracker for snapshot info.

Reply:
lets confirm with @kotreshhr too

Reply:
@humblec above tracker is not enough? what exact confirmation you are looking for from @kotreshhr here?

Reply:
> I don't think the above statement is correct. in the initial version snapshot info was added PTAL https://github.com/ceph/ceph-csi/pull/394/files#diff-e2d8af0e47a4dfb1dab79c4f16b787abbf3a5f94ebf4e8a9d5f41805246666f5R107-R138

you mean this is not correct https://github.com/ceph/ceph-csi/pull/2732#discussion_r772922760 ? In this comment I was pointing out `snapshot info` along with `subvolume info` was part of the PR, which is correct :)

Reply:
>so atleast in the initial implementaion it was only subvolume info we required and later snapshot info was added which added dependency to ceph version 15.2.4 may be`

i meant to say The above statement is not correct. both commands are required when it got implemented

Reply:
@Madhu-1 thats what I said in this comment too  https://github.com/ceph/ceph-csi/pull/2732#discussion_r772922760 ?

Reply:
@humblec the comment came in parallel when I was replying IMO we don't need to discuss it.

Reply:
ok... at glance  of the initial implementation PR, it looks like this is what happened, it was qualified against 15.2.3 where subvolume info was part of and its required. The initial implementation also made use of `snapshot info` , but it looks like there was an escape path of the code flow ie `checkSnapExist->Checkreservation` https://github.com/ceph/ceph-csi/pull/394/files#diff-642751ced65deb625b9d5cc2ba010870544ae4bcada00d4420aad6adbc5cfd6fR331 which could be the reason for the initial implementation test to have passed even with `15.2.3` .. @Madhu-1 wdyt? if something else caused the initial implementation PR to be passed in 15.2.3 or versions < 15.2.4 , please share your thought.

Reply:
I don't know if it's ever tested with octopus <=15.2.3 .please let me know what else is required from my side on this PR.

Reply:
@humblec @Madhu-1 As discussed by you guys, 

1. `subvolume info` is available from 15.2.2
2. `snapshot info` is available from 15.2.4

Reply:
Thanks @kotreshhr for confirming! ..